BrowserKit/Sources/ToolbarKit/ToolbarElement.swift

```diff
@@ -26,6 +26,8 @@ public struct ToolbarElement: Equatable {
     /// Indicates if the element should be displayed as highlighted
     let shouldDisplayAsHighlighted: Bool
 
+    let hasCFR: Bool?
```

Comment:
+1 or even provide a better naming convention for the variable.  

An example above, I prefer to have `let numberOfOpenTabs: Int?` since it describes the variable in its name and no need to add a comment.